installer/mac/ChainCore/ChainCore.swift

```diff
@@ -129,7 +129,8 @@ class ChainCore: NSObject {
         task.environment = [
             "DATABASE_URL": databaseURL,
             "LISTEN":       ":\(port)",
-            "LOGFILE":      self.logURL.path
+            "LOGFILE":      self.logURL.path,
+            "USER":         NSUserName(),
```

Comment:
Can we put a comment here to indicate why it's necessary? I think its role is a little less obvious than the other envvars.